core-web/.prettierrc

```diff
@@ -6,6 +6,7 @@
     "trailingComma": "none",
     "arrowParens": "always",
     "bracketSameLine": true,
+    "htmlWhitespaceSensitivity": "ignore",
```

Comment:
@oidacra Why do you ignore this rule?

Reply:
I have an issue with the `>` at the end of a closing tag; the prettier put in a new line.